pom.xml

```
@@ -386,11 +387,47 @@
           </configuration>
           <executions>
             <execution>
-              <id>integration-tests</id>
+              <id>integration-tests-altnet</id>
```

Let's change this to align with the naming used in code (also, most humans will call this `testnet` IMHO).

```suggestion
              <id>integration-tests-testnet</id>
```